composer/bin/mago

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
```

Comment:
I've kept this file for a few reasons:

* Composer adds a binary proxy wrapper. It somehow expects this to be a PHP file.
* In windows, the executable will be suffixed with .exe. I'm not sure if it runs without that suffix.
* This gives us some control about the availability of the executable. For example when composer runs with `no-scripts`